pkg/metadata/gohai/gohai.go

```diff
@@ -65,3 +74,12 @@ func getGohaiInfo() *gohai {
 
 	return res
 }
+
+var docker0Detector = sync.Once{}
```

Comment:
nit: I think declaring simply `var docker0Detector sync.Once` is more idiomatic way of obtaining the zero value of a struct.

Reply:
True, done